explorer/.env.dev

```
@@ -25,3 +25,6 @@ MAX_BATCH_SIZE=268435456 # 256 MiB
 
 # Time we wait for a batch to be verified before marking it stale
 BATCH_TTL_MINUTES=5
+
+# Latest aligned release that operators should be running 
+LATEST_RELEASE=v13.0.0
```

The latest version is v0.13.0
```suggestion
LATEST_RELEASE=v0.13.0
```

---

Solved in [ee6be8a](https://github.com/yetanotherco/aligned_layer/pull/1738/commits/ee6be8a435893fe2c1584f3ae70e62076b783840)